examples/colab/Bangla_article_classifier.ipynb

```diff
@@ -0,0 +1,1000 @@
+{
```

Comment:
<p>If you want, I think it would make sense to highlight this is a summer of code contribution, for example by adding a sentence at the top of the paragraph:</p><p> </p><p><em>Created by @Tahsin-Mayeesha for Google Summer of Code 2019</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>